src/main/webapp/WEB-INF/spring/initialize-beans.xml

```diff
@@ -138,23 +138,13 @@
 			</list>
 		</property>
 
-		<property name="mapConfigs">
-			<list>
-				<ref local="stdMapConfig" />
-			</list>
-		</property>
+		<property name="mapConfigs" ref="stdMapConfig" />
```

Comment:
No it won't. I erroneously removed the definition if the list, sry. :confounded:

Reverted this in 59385c0